app/config/question-bank.js

```diff
@@ -499,30 +514,102 @@ module.exports = {
+                  <li>dosing equipment</li>
+                  <li>mixing tank</li>
+                  <li>pump</li>
+                </ul>
+                `
+              },
               mustSelect: true,
-              errorMustSelect: 'This option must be selected -- !!'
+              errorMustSelect: 'You must select mild acidification equipment'
             },
             {
               key: 'project-items-A2',
               value: 'Acidification infrastructure',
-              hint: ''
-            },
-            {
-              key: 'project-items-A3',
-              value: 'Slurry pipework'
+              hint: {
+                text: 'Any work to adapt or install pipework, pumps etc to get slurry into the acidification system and then out to storage.'
+              }
             }
           ],
           yarKey: 'projectItems'
 
         },
+        {
+          key: 'slurry-application',
+          order: 81,
+          title: 'Slurry application - title',
+          pageTitle: '',
+          url: 'slurry/slurry-application',
+          baseUrl: 'slurry-application',
+          backUrl: 'project-items',
+          nextUrl: 'project-cost',
+          eliminationAnswerKeys: '',
+          ineligibleContent: {
+            messageContent: '',
+            insertText: { text: '' },
+            messageLink: {
+              url: '',
+              title: ''
+            }
+          },
+          fundingPriorities: '',
+          type: 'multi-answer',
+          minAnswerCount: 1,
+          maxAnswerCount: 3,
+          hint: {
+            html: `
+              Slurry Application - hint`
+          },
+          ga: { dimension: '', value: '' },
+          validate: {
+            errorEmptyField: 'Select all the items your project needs'
+          },
+          validations: [
+            {
+              type: '',
+              error: '',
+              regEx: '',
+              dependentAnswerKey: ''
+            }
+          ],
+          answers: [
+            {
+              key: 'project-items-A1',
+              value: 'Mild acidification equipment',
+              hint: {
+                html: `<span>You must buy all 4 of the following items:</span>
+                <ul>
+                  <li>acid storage</li>
+                  <li>dosing equipment</li>
+                  <li>mixing tank</li>
+                  <li>pump</li>
+                </ul>
+                `
+              },
+              mustSelect: true,
+              errorMustSelect: 'You must select mild acidification equipment'
+            },
+            {
+              key: 'project-items-A2',
+              value: 'Acidification infrastructure',
+              hint: {
+                text: 'Any work to adapt or install pipework, pumps etc to get slurry into the acidification system and then out to storage.'
+              }
+            }
+          ],
+          yarKey: 'slurryApplication'
+        },
```

Comment:
do you mind just deleting all of this informations for this page and put empty strings instead of these..it makes it easier and more clear more who wants to work on this question